stable/redis-ha/templates/redis-sentinel-service.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
     name: {{ template "name" . }}-sentinel-svc
     role: service
 {{ include "labels.standard" . | indent 4 }}
-  name: redis-sentinel
+  name: {{ .Release.Name }}-redis-sentinel
```

Comment:
I would suggest `{{ template "fullname" . }}-sentinel` as in the Sentinel Deployment, wdyt?

Reply:
Yes agreed. Updated